src/Components/3DV/SceneView.tsx

```diff
@@ -372,10 +372,14 @@ function SceneView(props: ISceneViewProps, ref) {
                         );
 
                         camera.attachControl(canvas, false);
-                        camera.lowerRadiusLimit = 0;
                         cameraRef.current = camera;
                         cameraRef.current.zoomOn(meshes, true);
```

Comment:
Not sure if it helps, but if you set the zoomOn bool to false the camera will auto adjust the maxZ to include the bounds of the object.

Reply:
You can set the bool on zoomOn, but that then clips when the user zooms out, so we need to set the MaxZ to be higher than the upperRadiusLimit. Updated code to make it clearer what its doing.